### PR TITLE
[Hotfix][seatunnel-common] fix may cause error when listFiles return null

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

File: seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java [hunk 1]
```diff
@@ -105,20 +105,20 @@ public static Long getFileLineNumber(@NonNull String filePath) {
...
     public static Long getFileLineNumberFromDir(@NonNull String dirPath) {
         File file = new File(dirPath);
-        Long value = null;
         if (file.isDirectory()) {
-            value = Arrays.stream(file.listFiles()).map(currFile -> {
+            File[] files = file.listFiles();
+            if (files == null) {
+                return 0L;
+            }
+            return Arrays.stream(file.listFiles()).map(currFile -> {
```
Comment (zhuangchong) [inline seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java]: Here `file.listFiles()` suggests to use the above variable `files`.

**Review by zhuangchong — CHANGES_REQUESTED** [review]

**Review by hailin0 — DISMISSED** [review]
update title: [Bug][seatunnel-common] Fix may cause error when listFiles return null

File: seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java (hunk 1 above)
Comment (CalvinKirs) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java]: +1

**Comment by TyrantLucifer** [comment]
> update title: [Bug][seatunnel-common] Fix may cause error when listFiles return null

+1, it might change `Bug` to `Hotfix` be better.

**Comment by EricJoy2048** [comment]
> > update title: [Bug][seatunnel-common] Fix may cause error when listFiles return null
> 
> +1

Thanks.
How can we tell when to use a `bug` and when to use `HotFix`?

File: seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java (hunk 1 above)
Comment (EricJoy2048) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/utils/FileUtils.java]: done

**Review by CalvinKirs — DISMISSED** [review]

**Review by zhuangchong — DISMISSED** [review]

**Review by liugddx — APPROVED** [review]

**Review by TaoZex — APPROVED** [review]

**Comment by TyrantLucifer** [comment]
> > > update title: [Bug][seatunnel-common] Fix may cause error when listFiles return null
> > 
> > 
> > +1
> 
> Thanks. How can we tell when to use a `bug` and when to use `HotFix`?

I think use `Hotfix` always.

**Review by Hisoka-X — APPROVED** [review]

**Review by TyrantLucifer — APPROVED** [review]
+1

**Pull request merged**